sitemap.xml

```diff
@@ -0,0 +1,189 @@
+<?xml version="1.0" encoding="utf-8"?><!--Generated by Screaming Frog SEO Spider 14.3-->
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1">
+  <url>
+    <loc>https://flyte.org/</loc>
+    <lastmod>2021-03-17</lastmod>
+    <changefreq>monthly</changefreq>
+    <priority>1.0</priority>
+  </url>
+  <url>
+    <loc>https://docs.flyte.org/projects/flytekit/en/latest/</loc>
+    <lastmod>2021-03-31</lastmod>
+    <changefreq>monthly</changefreq>
```

Comment:
Can we make this daily?

Reply:
Sure. I auto-generated the sitemap.xml file. So it's populated as monthly.